.travis.yml

```diff
@@ -12,5 +12,17 @@ cache:
 install: true
 
 script:
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-common/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-jodatime/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-web/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-string/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-validator/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0201/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0208/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0208kanji/pom.xml formatter:validate
+  - mvn -f terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0213kanji/pom.xml formatter:validate
```

Comment:
new version formatter-maven-plugin can not execute with above command .
we should execute per module( per pom.xml) with above command .
But I will change to below 1 line command. This commad search pom.xml and execute `formatter:validate` per module(per pomx.ml)
```
find ./terasoluna-gfw-common-libraries -name "pom.xml" -type f -exec mvn formatter:validate -f {} \; 
```